scripts/quests/outlands/Black_Market.lua

```diff
@@ -0,0 +1,139 @@
+-----------------------------------
+-- Black Market
+-----------------------------------
+-- Log ID: 5, Quest ID: 130
+-----------------------------------
+local quest = Quest:new(xi.questLog.OUTLANDS, xi.quest.id.outlands.BLACK_MARKET)
```

Comment:
Can you please ensure that you're commenting relevant NPCs and position values for testing this quest? (see other quest formatting)